docker/docker_client.go

```diff
@@ -103,11 +103,47 @@ func newDockerClient(ctx *types.SystemContext, ref dockerReference, write bool)
 		tlsc := &tls.Config{}
 
 		if ctx.DockerCertPath != "" {
-			cert, err := tls.LoadX509KeyPair(filepath.Join(ctx.DockerCertPath, "cert.pem"), filepath.Join(ctx.DockerCertPath, "key.pem"))
```

Comment:
So, this changes behavior. Are there any existing users (of `skopeo --cert-path` primarily) we need to worry about?